### DISCUSSION TITLE
Contributor call: 23 July 2022

### DISCUSSION THREAD

**Discussion by l3xander** [freesewing/freesewing#2415] [Support] [answered]
You are all invited to the FreeSewing Contributor Call.
Please leave a comment here or [on Discord](https://discord.freesewing.org/) to add things to the agenda.
This issue will be updated after the call with notes and to-dos.

### About
FreeSewing's contributor call is an informal hangout/chat/meeting that is open to all FreeSewing contributors and users alike.
The call is organized/hosted by @tangerineshark and @kaermorhan.
The call does not have a fixed structure or agenda items. Everyone is free to suggest agenda items, ask questions, or bring ideas.

Details:
Location: https://discord.freesewing.org (meeting room: 🔊voice chat)
Date: 23 July 2022 4pm UTC, which can be found in your local time zone [here](https://savvytime.com/converter/utc-to-belgium-antwerpen-ca-los-angeles-il-chicago-ny-new-york-city-brazil-sao-paulo-united-kingdom-london-france-paris-south-africa-johannesburg-msd-india-mumbai-hkt-japan-tokyo-australia-sydney/jul-23-2022/4pm).
Or check the bot in Discord! #meetings
The meetings will always be at 6pm, Antwerp time.

### Agenda
- `create-freesewing-pattern` is deprecated in favor of `@freesewing/new-design` , this also means we were able to drop many dependencies from the monorepo
- `remark-jargon` is broken in mdx 2 and will be replaced with `rehype-jargon`
- Babel is no longer a monorepo dependency, or used in tests (required wom changes in `plugin-theme`)
- Issues with crowdin and translation to new languages
- @joostdecock now runs our Instagram account (again)
- anyone interested in note-taking contributor calls?
- (if time) @tangerineshark has a question regarding some rectangles

> This discussion will be updated after the call with notes and to-dos.

**Comment by KaerMorhan** [comment, marked as answer]
**Notes**

Participants:

- @joostdecock 
- @tangerineshark 
- @bobgeorgethe3rd 
- @nataliasayang
- @eriese
- @SeaZeeZee
- @lexander2002 
- @bbdapalkow
- @KaerMorhan

**Joost**

- Note on the deprecated channels that advises to use other channels instead.
- _**remark-jargon**_ is broken in mdx 2 and will be replaced with _**rehype-jargon**_ (sorry, but I have no idea what that means)
- Babel is not working anymore, JavaScript will be used instead.
- New language on Crowdin – Ukrainian, being slowly translated by @bbdapalkow and @KaerMorhan :)

**Karen&Joost**

- FreeSewing Instagram account continues to run thanks to @lexander2002 and @joostdecock since @eleonoremilbou is resining. Any help and ideas are appreciated.
- Reddit is a bit dormant as well. Something to improve upon later.
- @lexander2002 is unavailable for taking notes for the nearest time so if somebody wants to fill that position – you’re more than welcome. If not, it’s also cool – there’re always @KaerMorhan and @tangerineshark.

**Eriese**
Updating on improving UX regarding the fabric layout, specifically moving around pieces on the screen to achieve desirable position for future fabric cutting. Extra buttons, rotating tool. Cutting information inside the pattern drafting.
For now, it works on the unfolded fabric. Can help with understanding how much fabric a user needs. Cutting instructions on the pieces themselves.
✨ FuckTrees ✨ mode that shows every pattern piece to better understand the total amount of fabric needed for sewing.
Another way down the line feature for cutting layout – some way to figure out how to line up for continuity on patterned fabrics.
_Bobgeorgethe3rd’s feedback: thinking about marking pattern pieces on fold, thinking about coding it, using several different fabrics.
Joost’s feedback: mentioning the code side of this, improving cutting instructions (grainline, cut on fold, new client-side PDF tiling etc)._

**Natalia**
Adding new designs to Strapi. Adding new showcases. If anybody is willing to join the Showcase Committee – again, you’re more than welcome.
Possibly introducing a bot to the process. Gathering data for structured showcasing through the bot (inviting to the form on the website). Keyboard shortcuts for doing the work.
_(Also: the tiniest Teagan for a new human.)_

**Bobgeorgethe3rd**
Suggesting locking messages on the welcome channel. A message that guides newcomers to the right places to seek help (maybe a bot also).

**Reply by joostdecock** [reply]
Thank you :)

**Reply by joostdecock** [reply]
PS: Eriese is @eriese

**Reply by joostdecock** [reply]
Natalia is @nataliasayang

**Reply by KaerMorhan** [reply]
Thank you, notes updated! :)

**Reply by eriese** [reply]
Another way down the line feature for cutting layout: some way to figure out how to line up for continuity on patterned fabrics

**Comment by joostdecock** [comment]
Closing to increase visibility of active discussions
